Author: Andreas Beckmann <anbe@debian.org>
Description: fix FTBFS with GCC 10
 GCC 10 defaults to -fno-common

--- a/libexplain/output/stderr.h
+++ b/libexplain/output/stderr.h
@@ -22,7 +22,7 @@
 
 #include <libexplain/output.h>
 
-explain_output_t explain_output_static_stderr;
+extern explain_output_t explain_output_static_stderr;
 
 #endif /* LIBEXPLAIN_OUTPUT_STDERR_H */
 /* vim: set ts=8 sw=4 et : */
